packages/rancher-monitoring/rancher-pushprox/charts/Chart.yaml

```diff
@@ -11,4 +11,4 @@ description: Sets up a deployment of the PushProx proxy and a DaemonSet of PushP
   clients.
 name: rancher-pushprox
 type: application
-version: 0.1.6
+version: 102.0.2
```

Comment:
This should be 103.x.x since it's being merged in the 2.8 release line

Reply:
Oops yes, I'll update this thanks!